corehq/apps/hqwebapp/utils/bootstrap/changes.py

```diff
@@ -41,6 +41,10 @@ def _get_plugin_regex(js_plugin):
     return r"(\.)(" + js_plugin + r")(\([\{\"\'])"
 
 
+def _get_extension_regex(js_plugin):
+    return r"(\$\.fn\.)(" + js_plugin + r")(\.Constructor)"
```

Comment:
Why are groups used in this regex? It looks like they are not referenced anywhere, and it's easier to read without them.
```suggestion
    return r"\$\.fn\." + js_plugin + r"\.Constructor"
```

I realize the answer may be "Because other similar expressions use them." which begs the question, could they be removed from the other expressions as well?

Reply:
i feel it's easier to read with groups but that's me 😅 and I was using the group details in some of the other regexs. kept the pattern around for consistency and my brain preferred it